File: URI/uri1013.py
print "URI 1013"
a, b, c = [int(i) for i in raw_input("").split(" ")]
MaiorAB = (a + b + abs(a-b))/2
MaiorABC = (MaiorAB + c + abs(MaiorAB - c))/2
print MaiorABC, "eh o maior"

File: Java/URI/uri1013.java
import java.util.Scanner;

public class uri1013{
	public static void main(String[] args){
		Scanner scanner = new Scanner(System.in);
		int a = scanner.nextInt();
		scanner.nextLine();
		int b = scanner.nextInt();
		scanner.nextLine();
		int c = scanner.nextInt();
		scanner.nextLine();

		int maiorAB = (a+b+Math.abs(a-b))/2;
		int maiorABC = (c+maiorAB+Math.abs(c-maiorAB))/2;

		System.out.format("%d eh o maior", maiorABC);
	}
}